### ISSUE TITLE
use "node" conditional exports to export ImageMagick that auto-initializes magick-wasm

### ISSUE THREAD

**Issue by iambumblehead** [dlemstra/magick-wasm#167]
### Is your feature request related to a problem? Please describe.

The current ImageMagick behaviour is "fine" but involves some ceremony loading and initializing magick.wasm.

### Describe the solution you'd like

For node environments (and deno), it would be nice if ImageMagick would load and initialize "magick.wasm" itself. Scripts using ImageMagick could then use ImageMagick without any magick.wasm load-init ceremony.

https://nodejs.org/api/packages.html#conditional-export

Basically, to do this, the '@imagemagick/magick-wasm' package would export a node-specific file,
```diff
        ".": {
            "import": "./dist/index.js",
+           "node": "./dist/index.node.js",
            "require": "./dist/index.umd.cjs",
            "types": "./dist/index.d.ts"
        },
```

Then, inside the node-specific file, init and load "magick.wasm" wherever is best

### Additional context

thank you :)

**Comment by dlemstra** [comment]
I don't think that I want to add that. I would prefer it if the browser and node api remain similar to make it easier to test/document things.

**Comment by iambumblehead** [comment]
Politely closing this un-wanted issue. Thank you for considering it.

**Comment by iambumblehead** [comment]
It would be nice if magick.wasm-file-loading were simplified for all environments. Maybe the current interface could still be reduced to a simpler call such as this
```javascript
MagickWasm.init(import.meta.resolve('@imagemagick/magick-wasm/magick.wasm'))
```

and maybe the wasm moduleId or url are optional, helping things to be smaller still
```javascript
MagickWasm.init()
```

**Comment by dlemstra** [comment]
I also don't think I want to do that now. Maybe in the future but I also don't think it's such a hassle to use the current api.

**Comment by lrowe** [comment]
I think this is worth reconsidering this since the included deno example only works from a checkout of this repository (`const wasm = await Deno.readFile('../dist/magick.wasm');`) and it does not seem to be possible to run from an npm import (`import * as im from "npm:@imagemagick/magick-wasm";`) since it seems impossible to resolve the `nom:@imagemagick/magick-wasm/magick.wasm` export from outside of the `@imagemagick/magick-wasm` package.
